deployments/gcp/multi-region/terraform.tfvars.sample

```diff
@@ -72,7 +72,7 @@ centos_admin_ssh_pub_key_file = "~/.ssh/id_rsa.pub"
 # secrets. The GCP service account specified above in "gcp_service_account"
 # must be in the same project and have KMS decryptor permissions for this key.
 
-kms_cryptokey_id = "projects/<project-id>/locations/<location>/keyRings/<keyring-name>/cryptoKeys/<key-name>"
+kms_cryptokey_id  = "projects/<project-id>/locations/<location>/keyRings/<keyring-name>/cryptoKeys/<key-name>"
```

Comment:
I just realized I didn't fix this, not supposed to be an extra space here. Should I recommit and force push?